vizro-core/examples/scratch_dev/assets/css/custom.css

```
@@ -1,8 +1,29 @@
-#page-header {
-  padding-left: 8px;
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
 }
 
-#old-line-height p {
-    line-height: 1;
-    margin-bottom: 0.5rem;
+.bg-container {
+  border: 1px solid var(--border-subtleAlpha01);
+  padding: 1rem;
+}
+
+.collapsible-container-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding-right: 12px;
+}
+
+#collapsible-container_icon.material-symbols-outlined {
```

```suggestion
.collapsible-container-header .material-symbols-outlined {
```
Otherwise the collapse icons look different in the first collapsible container compared to the second